app/styles/app.scss

```diff
@@ -128,30 +114,27 @@
   padding-top: 30px;
   padding-bottom: 30px;
   border-bottom: 1px solid #dddddd;
-  color: #333333;
+  color: black;
```

Comment:
@laurenbarker, Do you happen to have hex for the darkest acceptable color?
@sheriefvt Black is generally too much contrast. A *very* dark grey is more appealing to the eye.

Reply:
It's `#333333`.